bin/content_server_monitor

```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I can't say for sure, but I don't think this is being used anymore.

Reply:
Isn't this is how the metadata wrangler gets the initial information about Gutenberg books, so it can go off to OCLC and do a title-author search to get more information?

I think we can definitely get rid of this script in the medium term, but I don't know how else we'd learn the title/author of `http://www.gutenberg.org/etext/863` right now.

Reply:
Or is this now `LookupClientCoverageProvider`'s job?

Reply:
`DetailedOPDSImporter` doesn't exist in core anymore, so I don't think this script is being used. It would be `LookupClientCoverageProvider`'s job now, but this does beg the question of order. I'm going to make sure that OCLC Classify gives a transient failure status to books that don't have a title and author.

Reply:
It does. So content server works won't be run through OCLC Classify until we get the bibliographic information from the LookupClientCoverageProvider.